src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  margin-top: 16px;
+  margin-bottom: 24px;
```

Comment:
There are multiple `margin` properties set for the `.stars` block. It's better to combine them into a single property to follow the DRY principle. For example, `margin: 16px 0 24px;` combines the top and bottom margin into one declaration.